### PR TITLE
144 us 9 part 2

### PR DESCRIPTION
**Pull request by brilzen**

### CODE REVIEW

**Comment by brilzen** [comment]
tests added and completed 
image

File: client/scrapescholar_client/app/components/SearchView/CommentsSidebar.tsx
Comment (beedawn) [inline client/scrapescholar_client/app/components/SearchView/CommentsSidebar.tsx:1]: I'm confused by lines 41-44.

Shouldn't it be comments length >= 100 ?

I think we should also add a check on the post end point for the backend to check if comments are greater than 100. I am adding these and pushing a commit to this branch.

Comment (brilzen) [reply client/scrapescholar_client/app/components/SearchView/CommentsSidebar.tsx:1]: Apologies that was my test value I was testing with 2 in order to see if I get the error message.
I was about to change that and you beat me to it thank you good catch

**Comment by brilzen** [comment]
Thanks @beedawn for adding the backend logic also

**Review by beedawn — APPROVED** [review]
nice work!

**Review by tristanhamid — APPROVED** [review]

**Pull request merged**